plugins/logs/plugin.go

```diff
@@ -186,9 +188,12 @@ func (p *Plugin) Log(ctx context.Context, decision *server.Info) {
 
 	p.mtx.Lock()
 	defer p.mtx.Unlock()
-	dropped := p.buffer.Push(buf.Bytes(), false)
-	if dropped > 0 {
-		p.logInfo("Dropped %v events from buffer. Reduce reporting interval or increase buffer size.")
+
+	// compress the event
+	_, err := p.enc.Write(buf.Bytes(), p.buffer)
```

Comment:
Per comment above, I'd move the buffering back up to here and bring back the `plogInfo` message.